### PR TITLE
K8s:Added Encryption verticle   to RS Server

### PR DESCRIPTION
**Pull request by THARAK-RAM1**

### CODE REVIEW

**Review by pranavv0 — COMMENTED** [review]
LGTM

**Review by abhi4578 — CHANGES_REQUESTED** [review]
Generate Helm Charts READMEs' tables  based on values YAML file metadata using this tool - https://github.com/bitnami-labs/readme-generator-for-helm

eg: https://github.com/datakaveri/iudx-deployment/tree/master/K8s-deployment/Charts/resource-server#apiserver-parameters

File: K8s-deployment/Charts/resource-server/example-azure-resource-values.yaml [hunk 1]
```diff
@@ -80,7 +80,7 @@ databroker:
 encryption:
   enabled: true
   nodeSelector:
-    node.kubernetes.io/instance-type: t3.medium
+    node.kubernetes.io/instance-type: t3a.medium
```
Comment (abhi4578) [inline K8s-deployment/Charts/resource-server/example-azure-resource-values.yaml:83]: The nodeSelector should be node.kubernetes.io/instance-type: Standard_B2s
for azure

File: K8s-deployment/Charts/resource-server/README.md [hunk 2]
```diff
@@ -360,6 +360,70 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `encryption.sidecars`                              | Add additional sidecar containers to the encryption pod(s)                                           | `{}`                                                                                                                                                                                                                   |
+| `encryption.initContainers`                        | Add additional init containers to the encryption pod(s)                                              | `{}`                                                                                                                                                                                                                   |
+| `encryption.autoscaling.enabled`                   | Enable Horizontal POD autoscaling for Apache                                                         | `true`                                                                                                                                                                                                                 |
```
Comment (abhi4578) [inline K8s-deployment/Charts/resource-server/README.md:421]: It should be ``Enable Horizontal POD autoscaling for encryption``  and even all the remaining things related to autoscaling params

**Review by abhi4578 — CHANGES_REQUESTED** [review]
Also add async verticle parameters table

**Comment by abhi4578** [comment]
Also add async verticle parameters table @THARAK-RAM1

**Review by abhi4578 — APPROVED** [review]

**Pull request merged**